### ISSUE TITLE
PANDEMIC Machine throwing bluescreen when one virus cured/one virus uncured blood is inserted.

### ISSUE THREAD

**Issue by Stiix72** [tgstation/tgstation#65028]
Reporting client version: 514.1578

## Round ID:
[178695](https://scrubby.melonmesa.com/round/178695)

## Testmerges:
- [Nullspaces Some Undertile Objects (gas pipes/wires/disposals pipes) To Save On Maptick. For Real This Time](https://www.github.com/tgstation/tgstation/pull/63500)

## Reproduction:
I made a virus, someone came in with the virus I made and also the common cold. I cured the common cold, but my virus was still alive and well. 
The PANDEMIC machine bluescreened when the patient's blood was inserted (with one cured virus and one active virus).
See the error log posted below.

A fatal exception has occurred at 002B:C562F1B7 in TGUI. The current application will be terminated. Send the copy of the following stack trace to an authorized Nanotrasen incident handler at https://github.com/tgstation/tgstation. Thank you for your cooperation. 
 TypeError: Unable to get property 'symptoms' of undefined or null reference
   at s (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:1109138)
   at Ie (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:179474)
   at Anonymous function (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:179742)
   at Le (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:179720)
   at Te (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:180573)
   at Le (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:179584)
   at Anonymous function (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:179742)
   at Le (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:179720)
   at De (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:180768)
   at Te (http://127.0.0.1:53905/tmp14704/tgui.bundle.js:1:180607)
User Agent: Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; Tablet PC 2.0)
State: {"ckey":"stiix","interface":"Pandemic","window":{"key":"[0x200b111]-main","size":null,"fancy":1,"locked":1}}

Nanotrasen (c) 2525-2562

**Comment by JohnFulpWillard** [comment]
I have a hard time reading the reproduction instructions, you made a disease, got a DIFFERENT disease, cured it, got your own disease, then that cured blood is bluescreening the pandemic?

**Comment by LemonInTheDark** [comment]
Please stop writing your issue reports in the third person, it makes it hard to understand what you're saying. People who go through and fix these bugs are already rare, we shouldn't make their job any harder

**Comment by Stiix72** [comment]
Will write reports in first person in the future. Apologies for inconvenience.

**Comment by MrStonedOne** [comment]
> I have a hard time reading the reproduction instructions, you made a disease, got a DIFFERENT disease, cured it, got your own disease, then that cured blood is bluescreening the pandemic?

(original issue text for completeness:)

>Silkie made virus, someone comes in with the virus Silkie made and also common cold. Silkie cures common cold, but Silkie's virus is still alive and well. The PANDEMIC machine, when the blood of someone with one cured virus and one active virus is inserted, shows a bluescreen error, which is posted below.

This is perfectly understandable.

**Comment by optimumtact** [comment]
github is not a roleplay server MSO, and you are roleplaying as a giant asshole right now.

**Comment by MrStonedOne** [comment]
>can we please focus our efforts on productive uses of our time like actually fixing bugs and working on performance?

Only if @JohnFulpWillard actually starts reading issue reports instead of putting the blame on the op because they skimmed sentences instead of reading words.

**Comment by optimumtact** [comment]
It is okay to not understand something as long as you are polite in asking for clarification, and there's nothing in the tone of their comment that criticizes the OP for how they phrased it, just a comment that they found it hard to follow and the question mark indicates they are prompting them to weigh in with more info.

I understand that you understood the issue report with no problems, but that does not mean it's understandable to everyone.

It's not clear to me what you are taking issue with about John's comment, it does not appear disrespectful at all.

**Comment by jlsnow301** [comment]
#65050

**Comment by LemonInTheDark** [comment]
> Will write reports in first person in the future. Apologies for inconvenience.

I appreciate it, thanks. It's not a big inconvenience at all, no need to apologize.

**Comment by MrStonedOne** [comment]
> It's not clear to me what you are taking issue with about John's comment, it does not appear disrespectful at all.

It is my belief that the only way they could have not understood it, is if they didn't put in a good faith effort to actually read it. Which coming from an issue jannie, is Not A Great Look.

**Comment by optimumtact** [comment]
I don't think assuming they were acting in bad faith without trying to get more information out of either them or the PR creator first is a particularly good idea.
